CITING.md

```diff
@@ -119,15 +148,217 @@ articleno = {2},
 numpages = {26},
 keywords = {irregular matrices, GPUs, Sparse Matrix Vector Product (SpMV)}
 }
+
+@inproceedings{anztEvaluatingPerformanceNVIDIA2020,
```

Comment:
This paper is hard to put in one place, it's more than sparse to begin with, and also has solvers performance. It's essentially an A100 benchmarking paper, and comparing to V100 performance, I don't know if it fits here.